ui-v2/app/styles/components/list-collection.scss

```diff
@@ -3,5 +3,17 @@
   position: relative;
 }
 .list-collection > ul {
+  @extend %list;
+  border-top: 1px solid $gray-200;
   overflow-x: hidden !important;
 }
+%list > li:nth-child(2) .with-feedback p {
+  bottom: auto;
+  top: 24px;
+}
+%list > li:nth-child(2) p::after {
+  bottom: auto;
+  top: -10px !important;
+  border-bottom-width: 18px;
+  border-top-width: 0;
+}
```

Comment:
Also wondering if we should move all of the styles that relate to the tooltip whilst inside a `%composite-row` should be all together. You have some in here and some in `%composite-row`?

Reply:
no, %compositer-row is just the styling of a single row. Here I need to know the first item in list collection. I have to start outside of %composite-row.